libs/pageserver_api/src/models.rs

```diff
@@ -1257,6 +1256,10 @@ impl PagestreamFeMessage {
                     (request_lsn, request_lsn) // get version at specified LSN
                 }
             }
+            _ => (
```

Comment:
nit: I would prefer being explicit here (`PagestreamProtocolVersion::V2 | PagestreamProtocolVersion::V3`). We don't know how the wire format might evolve in the future.

Reply:
Updated